### ISSUE TITLE
Support full code inspection.

### ISSUE THREAD

**Issue by ChristianSchwarz** [arturbosch/detekt-intellij-plugin#34]
It should be possible to run an inspection on all files of a selected project, modul or package.

The results should be displayed like that one that is generated by Android Studio's "Inspect Code" action.

![grafik](https://user-images.githubusercontent.com/1437411/60898824-be54b900-a269-11e9-8451-e05ada3851af.png)

**Comment by sschuberth** [comment]
As a side note, that's actually the way I had expected the plugin to work. Neither the [README](https://github.com/detekt/detekt-intellij-plugin/blob/master/readme.md) nor the [plugin page](https://plugins.jetbrains.com/plugin/10761-detekt/) actually say how to execute a run of the detekt plugin. It took me a while to understand that it's seemingly meant to work as on-the-fly inspection hints, but that's a rather inconvenient mode of operation if you want to fix all detekt issues in bulk by working down a list.

**Comment by laithnurie** [comment]
Any update on this ?  like @ChristianSchwarz  mentioned, it will be very useful to have as an inspection action (that's what I imagined the plugin will do), or at least maybe create an inspection profile we can add ourselves manually.

@sschuberth did you manage to get something working for you ?

**Comment by sschuberth** [comment]
No, I simply stopped using the IntelliJ plugin in favor of the Gradle plugin :wink:

**Comment by JavierSegoviaCordoba** [comment]
I would like to replace native reformat code with this plugin so I would like to get:

- Reformat based on context (Selection, file, selected files, package, selected package, module, selected module and root for all.
- I developed with Dart (Flutter) and I love how the reformat works in that language, overall how it reformat the code if you use add/remove a trailing comma. I would like to get a similar behavior when Kotlin 1.4 get an estable versión.
